report/renderer/details-renderer.js

```diff
@@ -396,11 +396,23 @@ export class DetailsRenderer {
 
     const tbodyElem = this._dom.createChildOf(tableElem, 'tbody');
     let even = true;
+
     for (const item of details.items) {
       const rowsFragment = this._renderTableRowsFromItem(item, details.headings);
+
+      // The attribute item.entity could be a string (entity-classification), or
+      // a LinkValue for ThirdPartySummary audit.
+      let entityName;
```

Comment:
just checking, but this is the name/id (them being the same concept); not an index, right?

Reply:
Yeah this is expected to be a `string`, or unique name of entity. This can be used to look up the entity via `entityClassification.nameLUT` lookup table.